cleverpush/src/main/java/com/cleverpush/stories/models/StoryListModel.java

```diff
@@ -0,0 +1,34 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class StoryListModel {
+
+    @SerializedName("widget")
+    @Expose
+    private Widget widget;
+    @SerializedName("stories")
+    @Expose
+    private List<Story> stories = null;
+
+    public Widget getWidget() {
+        return widget;
+    }
+
+    public void setWidget(Widget widget) {
+        this.widget = widget;
+    }
+
+    public List<Story> getStories() {
+        return stories;
```

Comment:
please use this in return, as you want to reference to the class scope variable (make it more descriptive)